### ISSUE TITLE
False positive for operator `in` in pandas

### ISSUE THREAD

**Issue by ncarve** [microsoft/pylance-release#202]
## Environment data

-   Language Server version: 2020.7.4
-   OS and version: Ubuntu 18.04
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.7.7 (Anaconda)
-   Pandas version: 1.0.5

## Expected behaviour

Pylance should not detect any error when using the `in` operator on pandas Index objects (for example `'c1' in df.columns`)

## Actual behaviour

Pylance raises the following error when the `in` operator is used on a pandas Index object:

`Operator "in" not supported for types "str" and "Index[_str]"`

The code itself executes just fine. This happens for example when `in` or `not in` is used on a DataFrame's `columns` or `index` properties.

## Code Snippet / Additional information

```python
import pandas as pd

df = pd.DataFrame(columns=['c1', 'c2'], data=[[1, 2], [3, 4]], index=['i1', 'i2'])

if 'c1' in df.columns:
    print('ok')
if 'i1' in df.index:
    print('ok')
```

**Comment by erictraut** [comment]
@gramster, this is a bug in the type stub. It's missing the `__contains__` magic method in the `Index` class.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.8.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#202081-13-august-2020
